packages/retail-ui-extensions/src/extension-api/cart-api/cart-api.ts

```diff
@@ -35,7 +44,7 @@ export interface CartApiContent {
    * @param amount the percentage or fixed monetary amount deducted with the discount. Pass in `undefined` if using discount codes.
    */
   applyCartDiscount(
-    type: DiscountType,
+    type: DiscountType | CartDiscountType,
```

Comment:
Should we just remove `DiscountType` from here? What's the risk if we do that? AFAIK their IDE is just gonna show a linter error, even if they keep using the old type it will still run. But I feel like an error showing up might prompt them to migrate to the new type.

Reply:
Ooh, yeah, I think that makes sense. Depends on your definition of "breaking", I think.